### ISSUE TITLE
Staging Accessibility Review Findings for Display Facility Alerts

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#8221]
**Feedback framework**
- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

This epic will contain the accessibility staging review findings issue tickets.

## Point of Contact

**VFS Point of Contact:** Jennifer

cc @1Copenut — ready for VSP review, April 24, 2020, 3:30pm ET

**Comment by mmiddaugh** [comment]
Thank you for this feedback @jenstrickland. 

The following are important legacy items already prioritized for resolution. They were not part of this facility-level operating status implementation. 
- [#8245] 
- [#8243]

The following have been tagged for grooming and will be prioritized. 
- [#8344] 
- [#8414]

**Comment by jenstrickland** [comment]
@mmiddaugh — I still need for @1Copenut to review the audit, before it is complete.

**Comment by 1Copenut** [comment]
@jenstrickland && @mmiddaugh I'm reviewing right now. Should be done within an hour, possibly less.

**Comment by jenstrickland** [comment]
@mmiddaugh — accessibility findings complete.
